src/main/java/seedu/uninurse/model/medication/exceptions/DuplicateMedicationException.java

```diff
@@ -0,0 +1,11 @@
+package seedu.uninurse.model.medication.exceptions;
+
+/**
+ * Signals that the operation will result in duplicate Conditions.
+ * Conditions are considered duplicates if they have the same name.
```

Comment:
I think you forgot to change the JavaDoc comments
```suggestion
 * Medications are considered duplicates if they have the same name.
```